src/main/scala/io/qbeast/spark/internal/QbeastOptions.scala

```diff
@@ -33,13 +40,38 @@ object QbeastOptions {
     ColumnsToIndex.decode(encodedColumnsToIndex)
   }
 
+  /**
+   * Gets the desired cube size from the options
+   * @param options the options passed on the dataframe
+   * @return
+   */
+
   private def getDesiredCubeSize(options: Map[String, String]): Int = {
     options.get(CUBE_SIZE) match {
       case Some(value) => value.toInt
       case None => DEFAULT_CUBE_SIZE
     }
   }
 
+  /**
+   * Get the column stats from the options
+   * This stats should be in a JSON formatted string
+   * with the following schema
+   * {min(a):value, max(a):value, min(b):value...}
```

Comment:
IS this correct? In the tests the json is passed as `"min_columnName":10,"max_columnName":20`

Reply:
You are right! Thank you, I will change it for the correct format.